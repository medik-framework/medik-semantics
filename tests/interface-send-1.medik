interface ExternalMachine {
  var a;
}

init machine Test receives ExternalPing {
  init state Start {
    entry {
      var externalMachine = createFromInterface(ExternalMachine, "ex1");
      send externalMachine, EStart;
      exit;
    }
  }
}

