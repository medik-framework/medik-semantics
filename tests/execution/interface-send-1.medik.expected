{"id":"ex1","tid":1,"interface":"ExternalMachine","name":"EStart","args":[]}
